src/actions/index.js

```diff
@@ -0,0 +1,139 @@
+import {
+    GET_MOVIES_REQUEST,
+    GET_MOVIES_SUCCESS,
+    GET_MOVIES_ERROR,
+    SHOW_EDIT_MOVIE_PAGE,
+    HIDE_EDIT_MOVIE_PAGE,
+    SHOW_MODAL_ADD_MOVIE,
+    HIDE_MODAL_ADD_MOVIE,
+    SHOW_MODAL_EDIT_MOVIE,
+    HIDE_MODAL_EDIT_MOVIE,
+    SHOW_MODAL_DELETE_MOVIE,
+    HIDE_MODAL_DELETE_MOVIE,
+    HANDLE_ADD_MOVIE_FORM_FIELD,
+    SET_EDIT_MOVIE_ID,
+    REMOVE_EDIT_MOVIE_ID,
+    SET_DELETE_MOVIE_ID,
+    REMOVE_DELETE_MOVIE_ID,
+} from './types';
+
+const getMoviesRequest = () => {
+    return {
+        type: GET_MOVIES_REQUEST
+    };
+};
```

Comment:
You can write shorter
`const getMoviesRequest = () => ({ type: GET_MOVIES_REQUEST });`